controllers/nnf_workflow_controller.go

```diff
@@ -213,13 +213,14 @@ func (r *NnfWorkflowReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 
 		driverStatus.Status = dwsv1alpha1.StatusRunning
-		driverStatus.Message = ""
```

Comment:
I'd leave this since it is clear you are clearing the block of driver status values.  Then, for non-nil result, overwrite the message field. Same number of lines of code, but the organization makes it a little more obvious (for me)

Reply:
I'll put it back.